### PR TITLE
docs: Add OCM interaction information

### PR DESCRIPTION
**Pull request by miguelsorianod**
## Description
Adds information about
* How to interact with OCM
* Retrieve and generate OCM offline tokens and OCM tokens
* How to perform authenticated requests to the Fleet Manager API using OCM tokens
* How to setup Fleet Manager to be able to interact with OCM

## Checklist (Definition of Done)

- [ ] All acceptance criteria specified in JIRA have been completed
- [ ] Unit and integration tests added that prove the fix is effective or the feature works (tested against emulated and non-emulated OCM environment)
- [ ] Documentation added for the feature
- [ ] CI and all relevant tests are passing
- [ ] Code Review completed
- [ ] Verified independently by reviewer
- [ ] Required metrics/dashboards/alerts have been added (or PR created).
- [ ] Required Standard Operating Procedure (SOP) is added.
- [ ] JIRA has created for changes required on the client side

### CODE REVIEW

File: docs/populating-configuration.md [hunk 1]
```diff
@@ -18,17 +18,62 @@ to test things out.
...
 
-[OpenShift Cluster Managament](https://api.openshift.com/) is utilised for the management of Data plane clusters. 
-To be able to talk to this service, retrieve your ocm-offline-token from https://qaprodauth.cloud.redhat.com/openshift/token 
-and save and run the following command
+[OpenShift Cluster Management](https://api.openshift.com/) in Fleet Manager is
+used for mainly two purposes:
+* By Fleet Manager, to perform management of Data Plane clusters
+* By the users, to interact with the Fleet Manager API: OpenShift Cluster
+  Management (OCM) tokens are used as an Authentication mechanism to interact
+  with the Fleet Manager API
```
Comment (machi1990) [inline docs/populating-configuration.md:28]: `ocm token` is just a convenient way of getting a jwt token from sso.redhat.com, the default sso used for authentication: https://github.com/bf2fc6cc711aee1a0c2a/fleet-manager-go-template/blob/b705f061d0186e291741eedbeca8a4077c93be9b/pkg/server/server_config.go#L27-L29

so technically, a user does not need `ocm token` but rather a jwt token from sso.redhat.com. The ocm cli has just all of this connection mechanism built that's why we rely on it.

File: docs/populating-configuration.md [hunk 2]
```diff
@@ -18,17 +18,62 @@ to test things out.
...
+different kind of OCM tokens need to be generated and/or configured for each
+purpose. Keep reading below on how to do so.
+
+### Interacting with the Fleet Manager API
+
+The Fleet Manager API requires OCM Bearer tokens for authentication. The
+token is passed as an `Authorization` HTTP header value.
+
+To be able to perform requests to the Fleet Manager API, you first need
+to get an OCM token by requesting it to the OCM API. This can be done in an
```
Comment (machi1990) [inline docs/populating-configuration.md:39]: Based on what I've said on https://github.com/bf2fc6cc711aee1a0c2a/fleet-manager-go-template/pull/10/files#r803549522 I suggest we rephrase this to 
```suggestion
To be able to perform requests to the Fleet Manager API, you first need
to get a jwt token from sso.redhat.com, authentication server used by default. This can be done in an
```

**Review by machi1990 — APPROVED** [review]
added a comment and a small suggestion going in the way of that comment. Other than that it lgtm

File: docs/populating-configuration.md (hunk 1 above)
Comment (miguelsorianod) [reply docs/populating-configuration.md]: Reworded a little bit

File: docs/populating-configuration.md (hunk 2 above)
Comment (miguelsorianod) [reply docs/populating-configuration.md]: Reworded a little bit

**Review by machi1990 — APPROVED** [review]
Good to merge. Thanks
